provisioning/baremetal_pod.go

```diff
@@ -607,6 +607,7 @@ func createContainerMetal3IronicConductor(images *Images, config *metal3iov1alph
 			buildEnvVar(provisioningInterface, config),
 			buildSSHKeyEnvVar(sshKey),
 			setIronicHtpasswdHash(htpasswdEnvVar, ironicrpcSecretName),
+			setIronicExternalIp(externalIpEnvVar, config),
```

Comment:
I'm not clear why this isn't being done through buildEnvVar, but it matches the one for ironic-api.

Reply:
It looks like because there is some additional logic around Disabled provisioning network, and AFAICS the conditional in buildEnvVar only handles `PROVISIONING_IP` not `IRONIC_EXTERNAL_IP`

Maybe that can be refactored in future, but as you say this looks Ok for now?